modules/gitops-configuring-respect-rbac-using-gitops.adoc

```diff
@@ -0,0 +1,79 @@
+
+You can configure `respectRBAC` feature in the CLI.
+
+.Procedure
+
+. Create a YAML object file, for example, `argo-cd-resource.yaml`, to configure the `respectRBAC` feature using the CLI. 
++
+.Example create an Argo CD instance using the respectRBAC feature
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd #<1>
+spec:
+  controller:
+    respectRBAC: strict #<2>
+----
+<1> Specify the name of the Argo CD instance.
+<2> Specify the value of the `.spec.controller.respectRBAC` key in the Argo CD resource. You can set it to `normal` or `strict`. If the value is set to `strict`, Argo CD performs additional validations of RBAC resources to determine permissions. This setting increases the number of API calls to the server as it is more accurate compared to `normal`. Switch to `strict` if Argo CD reports errors indicating it cannot access resources when the value is set to `normal`. If the value is set to `normal`, Argo CD determines its RBAC permissions by attempting to list each resource. By default, the value is set to `normal`, as it balances accuracy and speed because resource listing is a lightweight operation.
+
+. Apply the changes to the YAML object file by running the following command.
++
+[source,terminal]
+----
+$ oc apply -f argocd-resource.yaml -n argo-cd-instance #<1>
+----
+<1> Specify the name of the YAML file that includes the Argo CD resource and the namespace that hosts the Argo CD instance.
++
+. Verify the successful installation of Argo CD by checking the status of the `argoCD.status.phase` field. The {gitops-shortname} Operator sets this field to `Available` when the installation is complete. Run the following command to check the status:
++
+[source,terminal]
+----
+kubectl get argocd <argocd-instance-name> -o jsonpath='{.status.phase}' #<1>
+----
+<1> Replace <argocd-instance-name> with the name of your Argo CD instance for example, `example-argocd`.
+
+[id="configuring-respectRBAC-using-the-web-UI_{context}"]
+== Configuring respectRBAC using the Web UI
+
+You can configure `respectRBAC` in the Web UI.
+
+.Procedure
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to install the user-defined Argo CD instance from the *Project* list.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
++
+After successful installation, the Argo CD instance is listed under *Argo CD* tab and its Status should be set to *Available*.
++
+. After the Argo CD instance is created, verify that the value in the Argo CD resource is updated in the *YAML* tab successfully by completing the following steps.
+
+.. In the *Administrator* perspective, go to *Workload* -> *ConfigMaps*.
+.. In the *Project* option, select *Argo CD* namespace.
+.. Select the *YAML* tab to view the `resource.respectRBAC` field.
```

Comment:
- ".. In the *Project* option, select *Argo CD* namespace."
- I think we are missing a step here, where user should select `argocd-cm` Configmap.
- ".. Select the *YAML* tab to view the `resource.respectRBAC` field."

Reply:
Hey I removed this step based on your earlier recommendation for config maps. Do we still need to retain this step @jgwest?

Reply:
Yes, this new step is required. Without this, there is no 'YAML' tab to view, IIRC, and thus the steps don't make sense.

Reply:
Ok, I have made the change as per your feedback. PTAL.